plugin/src/main/java/app/cash/paraphrase/plugin/ResourceWriter.kt

```diff
@@ -59,8 +60,35 @@ internal fun writeResources(
       """.trimIndent(),
     )
     .addImport(packageName = packageName, "R")
+    .addProperty(
+      PropertySpec.builder(
+        name = "AndroidFormattedResources",
```

Comment:
I went back and forth on how to name these:
* `BaseFormattedResources` and `FormattedResources` was Patrick's original suggestion, and allows for the simplest transition for current consumers. But it's the base class that most consumers would probably deal with, so giving it the longer name feels clunky. And having many different classes generated with the same prefix `Base` might be confusing since it is not a common base.
* `FormattedResources` and `AndroidFormattedResources` is what I've gone with. The generic type has the snappier name. But it makes the transition for current consumers more involved. And using the prefix `Android` might seem redundant for an Android-targeted library.
* Another idea: Rename `AndroidFormattedResources` to `FR`. This makes static invocations somewhat parallel to static invocations of the standard Android resource: `R.string.whatever` → `FR.whatever(args)`. But it's still an involved transition from the previous Paraphrase API. And maybe too cute.

Curious what others think for naming these two.

Reply:
I don't love the `Base` prefix either, I'm happy to go with either of the other two options. Another option might be a base of `ParaphraseResources` and implementations of `AndroidResources` and `JvmResources`.

I'm not sure there's a perfect naming scheme and I'm also not sure it's a big deal. I'm cool with just picking something and shipping it!

Reply:
I like `ParaphraseResources` because it steps around the verb-tense problem with "Formatted" and also reminds you what this thing is.

Reply:
That would also let me include a deprecated `val FormattedResources` with a built-in `ReplaceWith` to the new name, if we want to get fancy and make the transition easy.